### DISCUSSION TITLE
essay/2023/09/12/week-travel

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#128] [Comments]
# essay/2023/09/12/week-travel

가리봉동에서 태어나고… 초중고를 다 그 동네에서만 살았습니다. 서울에서 태어났지만 아직도 서울 지리를 잘 모릅니다.

https://jeho.page/essay/2023/09/12/week-travel.html

**Comment by kaynoh** [comment]
재미난 이야기에 좋은 정보 고맙습니다. 저도 주인장님처럼 뭔가 파이프라인을 얼른 하나 만들어야 할텐데 말이죠. ^^;

**Reply by BenjaminKim** [reply]
파이프라인 하나 만들어두시면 정말 좋습니다!

**Comment by soo-toance** [comment]
다양한 도시에서 모집자를 구하고 있군요 : )  시간 내서 한 번 살펴봐야겠어요 ! 감사합니다 !

**Reply by BenjaminKim** [reply]
너무 좋은 기회에요. 여건이 되시면 꼭 지원해보세요. 😁
